src/addon-manager-proxy.ts

```diff
@@ -161,7 +180,7 @@ class AddonManagerProxy extends EventEmitter {
    * @method onMsg
    * Called whenever a message is received from the gateway.
    */
-  onMsg(msg) {
+  onMsg(msg: any) {
```

Comment:
At first I tried to figure out type for `msg` param but then I noticed whole method is massive and based on message types generated dynamically. I think `onMsg()` is good candidate for refactor. Of course it is out of scope for this PR.